.github/workflows/reusable_testing.yml

```diff
@@ -0,0 +1,52 @@
+name: reusable_testing
+
+on:
+  workflow_call:
+    inputs:
+      artifact_name:
+        description: 'The name of the tarantool build artifact'
+        default: ubuntu-focal
+        required: false
+        type: string
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: 'Clone the expirationd module'
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.repository_owner }}/expirationd
+
+      - name: 'Download the tarantool build artifact'
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ inputs.artifact_name }}
+      # TODO(ylobankov): Just in case remove tarantool that could be previously
+      # installed on the runner. Relevant to self-hosted runners. Remove this
+      # step when only GitHub-hosted runners are used for integration testing.
+      - run: sudo apt-get -y purge 'tarantool*'
+      - name: 'Install tarantool'
+        # TODO(ylobankov): Install package dependencies. Now we're lucky: all
+        # dependencies are already there.
+        run: sudo dpkg -i tarantool*.deb
```

Comment:
I don't like `*` as a mask, let's specify what exactly we want to install. For example: `dpkg -i tarantool tarantool-dev tarantool-common`.

Reply:
It is the file glob: install what is extracted from the artifact.

Reply:
@ligurio Should we consider this discussion resolved?

Reply:
> It is the file glob: install what is extracted from the artifact.

I don't know what was extracted from the artifact.
Why we cannot run dpkg with package name and glob instead of version part?

Reply:
You mean `sudo dpkg -i tarantool_*.deb tarantool-common_* tarantool-dev_*`?

Reply:
I'm not against explicit package names (explicit is better than implicit, sure), but I would highlight my personal opinion about the tarantool-command package: I think it [should be removed](https://github.com/tarantool/tarantool/issues/6400) in a future.

`dpkg -i tarantool*.deb` will continue to work on all tarantool versions, but `dpkg -i tarantool_*.deb tarantool-common_*.deb tarantool-dev_*.deb` should be transformed into something like the following (after change in tarantool):

```sh
# gh-6400: blah blah.
if [ -f tarantool-common_*.deb ]; then
    dpkg -i tarantool-common_*.deb
fi
dpkg -i tarantool_*.deb tarantool-dev_*.deb
```

But I'll repeat that the linked issue reflects only my personal opinion (however I didn't heard the opposite opinions) and there are no certain plans to actually implement it.

Reply:
> You mean sudo dpkg -i tarantool_*.deb tarantool-common_* tarantool-dev_*?

Exactly.

Reply:
Ok, I don't mind. Fixed.